lib/tasks/emails.rake

```diff
@@ -0,0 +1,20 @@
+namespace :metamaps do
+  desc "delivers recent map activity digest emails to users"
+  task deliver_map_activity_emails: :environment do
+    summarize_map_activity
+  end
+
+  def summarize_map_activity
+    Follow.where(followed_type: 'Map').find_each do |follow|
```

Comment:
@devvmh I realized we can constrain the process to only call what needs to be called by looking for follows instead of for maps. I guess I might want to eager load the maps and users associations